rfcs/0019-storing-client-data.md

```diff
@@ -476,9 +476,39 @@ Coinbase also uses MongoDB to store all of the data in a blockchain.
 
 # Appendix
 
-The flexibility of this new design changed some parts of the current schema of the GraphqQL database. Here is the schema:
+The flexibility of this new design changed some parts of the current schema of the GraphqQL database. Here is the new proposed schema:
 
 ```graphql
```

Comment:
It's still not clear to me which parts of the schema changed -- so it's hard to review. Can you only include the parts that are different?